charts/spire/charts/spire-server/values.yaml

```diff
@@ -73,12 +73,28 @@ affinity: {}
 
 topologySpreadConstraints: []
 
-dataStorage:
+# Used by sqlite3 for spire-server, by the disk KeyStore, and/or by Tornjak.
+persistence:
   enabled: true
   size: 1Gi
   accessMode: ReadWriteOnce
   storageClass: null
 
+dataStore:
+  sql:
+    # -- Other supported databases are "postgres" and "mysql"
+    database_type: sqlite3
+    database: spire
+    host: ""
+    # -- likely 5432 for postgresql 3306 for mysql
+    port: 0
+    username: spire
+    password: ""
```

Comment:
Should the password be read from the secret?

Reply:
Are you asking for the option of reading the password out of an existing k8s secret?